isabelle/ROOT

```diff
@@ -10,7 +10,7 @@ session "TLA+" = "Pure" +
 
   document_files
     "root.tex"
-
+(*
```

Comment:
@muenchnerkindl, can I uncomment this?
I usually open the theories as `isabelle jedit -D isabelle/ isabelle/Constant.thy`, then all the modules see each other. I use these additional sessions in the ROOT file to build the test/examples theories during the tests.

Reply:
Sure, please go ahead. Made the change when I built the heap and forgot to revert it.